samples/readme.md

```diff
@@ -0,0 +1,120 @@
+# Plug-in Creation for Performance SDK
+
+This document will introduce two paths for creating Performance Toolkit SDK (SDK) plug-ins:
+
+1) Simple Data Source
+2) Extensible Data Source
+
+## Overview
+
+These two options offer a developer some flexibility in creating a plug-in.
+Below are some points of comparison.
+
+### Complexity
+
+A Simple Data Source requires less code to get started.
+
+An Extensible Data Source fits into a framework that requires additional code at the start of the project.
+
+### Data Accessibility
+
+Both options expose Tables which are requested when processing a data source.
+
+An Extensible Data Source offers programmatic access to data through Data Cookers (see below).
+
+Coming Soon: Both options support programmatic data access at the Table level.
+
+#### Data Cookers
+
+A Data Cooker consumes **raw data**, performs some processing, and exposes **cooked data**. This processing happens once during the lifetime of the Data Cooker and the cooked data does not change.
```

Comment:
>**raw data**, performs some processing, and exposes **cooked data** [](start = 23, length = 67)

It might be worthwhile to define "raw data" and "cooked data" here